src/modules/sdlog2/sdlog2_format.h

```diff
@@ -80,7 +80,7 @@ struct log_format_s {
 	uint8_t length;		// full packet length including header
 	char name[4];
 	char format[16];
-	char labels[64];
+	char labels[128];
```

Comment:
Not really without further inspection. Can you please revert and use shorter labels instead?